getting_started/srun.md

```diff
@@ -0,0 +1,55 @@
+## `srun`: Submitting Jobs to the Slurm Workload Manager
+
+`srun` is a command-line tool used for submitting jobs to the Slurm workload manager, offering a wide range of options for customizing job submission, and essentially letting you run parallel jobs on a cluster managed by slrum.
```

Comment:
Could you also link the slurm docs on [srun](https://slurm.schedmd.com/srun.html#index) somewhere in this as well.

Reply:
I'm sorry. Do you mean link srun to the slurm repository/docs?

Reply:
Yes!